### DISCUSSION TITLE
Snapdragon X elite

### DISCUSSION THREAD

**Discussion by yujikaido** [erew123/alltalk_tts#254] [General]
I just got my hands on my Samsung galaxy book 4 edge  with the new snapdragon x elite, running windows on arm will it be able to utilize any of the NPU or any of the new features of the chip?  Thank you, info is a bit sparse on running pytorch on the npu or on arm from what I have searched.

**Comment by erew123** [comment]
Hi @yujikaido I've certainly not added any new code that is ARM NPU specific and I've not seen anything with PyTorch that says is carries CUDA compatibility/emulation. I cant seem to see that on Qualcomms developer section either https://docs.qualcomm.com/bundle/publicresource/topics/80-63442-2/overview.html

So for now, I would say there is no immediate way to get any benefit from the NPU with the current TTS engines within AllTalk. 

Thanks

**Reply by yujikaido** [reply]
ok thank you for the reply, I guess i will stick with my Ubuntu install.
